frontend/src/scenes/session-recordings/player/sessionRecordingDataLogic.test.ts

```diff
@@ -208,6 +207,7 @@ describe('sessionRecordingDataLogic', () => {
         it('server error mid-fetch', async () => {
             const firstNext = `${EVENTS_SESSION_RECORDING_EVENTS_ENDPOINT}?person_id=1&before=2021-10-28T17:45:12.128000Z&after=2021-10-28T16:45:05Z`
             silenceKeaLoadersErrors()
+            jest.spyOn(api, 'get')
```

Comment:
Note: The changes in this file (outside of removing `RecordingWatchedSource`) clean up some test error logs that are un-related to this PR, but needed cleaning